runtime/src/error.c

```diff
@@ -79,6 +79,11 @@ static void spinhaltIfAlreadyExiting(void) {
 #endif
 }
 
+#define UNW_LOCAL_ONLY
```

Comment:
Can you put a comment explaining what UNW_LOCAL_ONLY does / why we need to define it?